sys/Makefile

```diff
@@ -82,23 +82,18 @@ endif
 ifneq (,$(filter tcp,$(USEMODULE)))
     DIRS += net/transport_layer/tcp
 endif
-
```

Comment:
yep - seems like I couldn't resist removing the asymmetry in this file when moving the diskio stuff^^